spring-cloud-task-core/src/main/java/org/springframework/cloud/task/listener/TaskMetrics.java

```diff
@@ -55,74 +63,80 @@
 	/**
 	 * task name measurement tag.
 	 */
-	public static final String TASK_NAME_TAG = "task.name";
+	public static final String TASK_NAME_TAG = "spring.cloud.task.name";
 
 	/**
 	 * task execution id tag.
 	 */
-	public static final String TASK_EXECUTION_ID_TAG = "task.execution.id";
+	public static final String TASK_EXECUTION_ID_TAG = "spring.cloud.task.execution.id";
 
 	/**
 	 * task parent execution id tag.
 	 */
-	public static final String TASK_PARENT_EXECUTION_ID_TAG = "task.parent.execution.id";
+	public static final String TASK_PARENT_EXECUTION_ID_TAG = "spring.cloud.task.parent.execution.id";
 
 	/**
 	 * task external execution id tag.
 	 */
-	public static final String TASK_EXTERNAL_EXECUTION_ID_TAG = "task.external.execution.id";
+	public static final String TASK_EXTERNAL_EXECUTION_ID_TAG = "spring.cloud.task.external.execution.id";
 
 	/**
 	 * task exit code tag.
 	 */
-	public static final String TASK_EXIT_CODE_TAG = "task.exit.code";
+	public static final String TASK_EXIT_CODE_TAG = "spring.cloud.task.exit.code";
 
 	/**
 	 * task status tag. Can be either STATUS_SUCCESS or STATUS_FAILURE
 	 */
-	public static final String TASK_STATUS_TAG = "task.status";
+	public static final String TASK_STATUS_TAG = "spring.cloud.task.status";
 
 	/**
 	 * task exception tag. Contains the name of the exception class in case of error or
 	 * none otherwise.
 	 */
-	public static final String TASK_EXCEPTION_TAG = "task.exception";
-
-	private Timer.Sample taskSample;
+	public static final String TASK_EXCEPTION_TAG = "spring.cloud.task.exception";
 
 	private LongTaskTimer.Sample longTaskSample;
 
-	private Throwable exception;
+	private Observation.Scope scope;
+
+	private TaskExecutionTagsProvider tagsProvider = new DefaultTaskExecutionTagsProvider();
+
+	private TaskExecutionObservationContext taskObservationContext;
 
 	public void onTaskStartup(TaskExecution taskExecution) {
-		LongTaskTimer longTaskTimer = LongTaskTimer
+		if (this.registry != null) {
+			this.taskObservationContext = new TaskExecutionObservationContext(taskExecution);
+			Observation observation = Observation.start(SPRING_CLOUD_TASK_METER, this.taskObservationContext, this.observationRegistry)
```

Comment:
You can have `ObservationRegistry` without `meterRegistry` theoretically. Just saying ;)